CustomBoomboxTracks/Managers/AudioManager.cs

```diff
@@ -111,12 +111,15 @@ private static IEnumerator WaitForAllClips(List<Coroutine> coroutines)
         public static void ApplyClips(BoomboxItem __instance)
         {
             BoomboxPlugin.LogInfo($"Applying clips!");
+            AudioClip[] audioClips = clips.ToArray();
+            Array.Sort(audioClips, delegate (AudioClip clip1, AudioClip clip2) { return clip1.name.CompareTo(clip2.name); });
 
             if (Config.UseDefaultSongs)
                 __instance.musicAudios = __instance.musicAudios.Concat(clips).ToArray();
             else
-                __instance.musicAudios = clips.ToArray();
+                __instance.musicAudios = audioClips;
 
+            __instance.musicRandomizer = new System.Random(12345);
```

Comment:
Is hard-coding the seed necessary? From what I understand from the game's code all boomboxes should share the seed of the level.

Reply:
It does. It may be out-of-sync when you test this in a pre-game lobby?

From BoomboxItem.cs:
```cs
public override void Start()
{
  musicRandomizer = new System.Random(playersManager.randomMapSeed - 10);
}
```